bokeh/models/renderers.py

```diff
@@ -242,6 +242,18 @@ def _check_malformed_graph_source(self):
     rendered as the graph edges.
     """)
 
+    selection_mode = Enum("default", "linked", help="""
+    With the "default" selection mode, node glyphs will be selected in the same
+    way as normal glyphs. With "linked" mode, selecting a node will also select
+    the attached edges.
+    """)
+
+    inspection_mode = Enum("default", "linked", help="""
+    With the "default" inspection mode, node glyphs will be inspected in the same
+    way as normal glyphs. With "linked" mode, inspecting a node will also inspect
+    the attached edges.
+    """)
```

Comment:
I we want to make these as flexible as possible, we might want to make these policies configurable with named models, e.g. `DefaultGraphSelection` and `LinkedGraphSelection` that would leave the door open for a `CustomJSGraphSelection` which I think would be a valuable relief valve for the inevitable fancy things people want to do that we can't maintain in the library proper